ex/high_throughput_inference/standalone_worker_manager.py

```diff
@@ -124,13 +119,15 @@ def service_as_dragon_proc(
         "--batch_size",
         type=int,
         default=1,
-        help="How many requests the workers will try to aggregate before processing them",
+        help="How many requests the workers will try "
+        "to aggregate before processing them",
     )
     parser.add_argument(
         "--batch_timeout",
         type=float,
         default=0.001,
-        help="How much time (in seconds) should be waited before processing an incomplete aggregated request",
+        help="How much time (in seconds) should be waited "
```

Comment:
Ditto on "how much time..." nitpick

```suggestion
        help="Maximum time allowed (in seconds) to aggregate samples "
             "before processing a partial batch",
```

Reply:
Thanks